CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.2.2
+
+* Use `Entity.to_json` method (see [commit on Core](https://github.com/openfisca/openfisca-core/commit/dd4d5df3b16ccdd77d5afaece608b13812484e9c))
```

Comment:
Changelogs should be less technical (I can see we are using the `to_json` method in the code) and more functional. What is bringing this change to the api ? Is it repairing a broken route ? Are we changing the JSON representation (and, if so, is it a breaking change ?).